swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -6679,7 +6810,7 @@ components:
         tppMessages:
           type: array
           items:
-            $ref: "#/components/schemas/tppMessage2XX"
+            $ref: "#/components/schemas/tppMessage201PaymentInitiation"
```

Comment:
Relevant Change: tppMessage201PaymentInitiation is new.
Adoption approach: Follow the change